presto-docs/src/main/sphinx/security/kerberos-configuration.fragment

```diff
@@ -0,0 +1,24 @@
+Kerberos Configuration
+^^^^^^^^^^^^^^^^^^^^^^
+
+Kerberos needs to be configured on the |subject_node| At a minimum, there needs
+to be a ``kdc`` entry in the ``[realms]`` section of the ``/etc/krb5.conf``
```

Comment:
I see references to KDC throughout. Is all in one "chapter"? If so, if it's defined at first mention, no need to define elsewhere.